Jenkinsfile

```diff
@@ -34,11 +34,12 @@ ansiColor('xterm') {
       }
     }
 
-    stage('Release unstable MoM EE Docker Image') {
-      if (env.BRANCH_NAME == 'master') {
+    stage('Release MoM EE Docker Image') {
+      if (env.BRANCH_NAME == 'master' || env.BRANCH_NAME ==~ /releases\/.*/) {
```

Comment:
It might break on `releases/1.9` because the plugin on master requires Scala 2.13.

Reply:
Why do you think it would? Wouldn't sbt fetch the version of Scala needed?

Reply:
Take a look a few lines down. All builds trigger the plugin releases of `master` which builds Marathon Plugins for Scala 2.13. We should move the plugins into this repo.